app/models/entities/Target.scala

```diff
@@ -218,17 +218,18 @@ object Target {
 
     implicit val safetyCodeImpW = Json.writes[models.entities.SafetyCode]
     implicit val safetyCodeImpR: Reads[models.entities.SafetyCode] =
```

Comment:
For reference: this commit addresses the issue I described in [this comment in pull request opentargets/platform-app#146](https://github.com/opentargets/platform-app/pull/146#discussion_r425934618).

Reply:
And when I re-try the query in the test deployment there, I can confirm that it does has indeed been addressed :+1: